app/views/admin/participants/validation_data/date_of_birth.html.erb

```diff
@@ -0,0 +1,19 @@
+<% title = "What is #{possessive_name(@validation_data_form.participant_name)} date of birth?" %>
+<% content_for :title, title %>
+<% content_for :before_content, govuk_back_link( text: "Back", href: admin_participant_path(id: @participant_profile)) %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl">Validation data - <%= @validation_data_form.participant_name %></span>
+    <h1 class="govuk-heading-xl"><%= title %></h1>
+    <%= form_for @validation_data_form, url: { action: :date_of_birth }, method: :put do |f| %>
+      <%= f.govuk_error_summary %>
+      <%= f.govuk_date_field :date_of_birth,
+        date_of_birth: true,
+        width: "two-thirds",
+        legend: { text: "#{possessive_name(@validation_data_form.participant_name)} date of birth", class: "govuk-visually-hidden" },
+        hint: { text: "For example, 14 7 1990"}
+      %>
```

Comment:
I think the form builder can be leant on more here, something along the lines of:

```suggestion
    <%= form_for @validation_data_form, url: { action: :date_of_birth }, method: :put do |f| %>
      <%= f.govuk_error_summary %>
      <%= f.govuk_date_field :date_of_birth,
        date_of_birth: true,
        width: "two-thirds",
        legend: { text: "#{possessive_name(@validation_data_form.participant_name)} date of birth", size: "xl", tag: "h1" },
        caption: { text: @validation_data_form.participant_name, size: "xl" },
        hint: { text: "For example, 14 7 1990"}
      %>
```

Reply:
Good idea.  The caption style was a suggestion from Brian from the previous set of admin pages I added to help keep the context of what you're doing.  I'll see how it looks.

Reply:
I think the suggestion definitely helps. All of the form builder field types (those with a label or legend) support it

https://govuk-form-builder.netlify.app/introduction/labels-captions-hints-and-legends/#a-label-with-a-caption